src/actions/write/tick.test.ts

```diff
@@ -977,6 +977,12 @@ describe('tick', () => {
           },
         },
       },
+      distributions: {
+        ...getBaselineState().distributions,
+        // setting these to next epoch values to validate distributions depend on only the nextDistributionHeight
+        epochEndHeight: SmartWeave.block.height + 2 * EPOCH_BLOCK_LENGTH - 1,
+        epochStartHeight: SmartWeave.block.height + EPOCH_BLOCK_LENGTH - 1,
+      },
```

Comment:
this is critical to this test setup - it would've caught the issue resulting in distributions not occurring.